sed/calibrator/delay.py

```diff
@@ -22,22 +28,33 @@ class DelayCalibrator:
 
     Args:
         config (dict, optional): Config dictionary. Defaults to None.
+        verbose (bool, optional): Option to print out diagnostic information.
+            Defaults to True.
     """
 
     def __init__(
         self,
         config: dict = None,
+        verbose: bool = True,
     ) -> None:
         """Initialization of the DelayCalibrator class passes the config.
 
         Args:
             config (dict, optional): Config dictionary. Defaults to None.
+            verbose (bool, optional): Option to print out diagnostic information.
+                Defaults to True.
         """
         if config is not None:
             self._config = config
         else:
             self._config = {}
 
+        self.verbose = verbose
+        if self.verbose:
+            logger.handlers[0].setLevel(INFO)
```

Comment:
It is not quite clear by [0] should be used.

Reply:
There is only one handler in each logger, which is always [0].